README.md

```diff
@@ -23,10 +23,10 @@ Explore the GitHub search queries used:
 
 ## Current Statistics
 
-| Project | Total PRs | Merged PRs | Merge Rate |
-| ------- | --------- | ---------- | ---------- |
-| Copilot | 16,721 | 7,457 | 44.60% |
-| Codex   | 299,655 | 252,892 | 84.39% |
-| Cursor  | 1,093 | 828 | 75.75% |
-| Devin   | 28,597 | 17,448 | 61.01% |
-| Codegen | 4,048 | 1,699 | 41.97% |
\ No newline at end of file
+| Project | Total PRs | Ready for Review PRs | Merged PRs | Success Rate |
+| ------- | --------- | -------------------- | ---------- | ------------ |
+| Copilot | 1,100 | 1,000 | 450 | 45.00% |
```

Comment:
These numbers are way lower.  We need to make sure e are still including all of the originally included PRs.  If you cannot access this data due to firewall restrictions - let me know.